mobile/test/common/jni/jni_helper_test.cc

```diff
@@ -0,0 +1,183 @@
+#include <jni.h>
+
+#include "library/common/jni/jni_helper.h"
+
+// NOLINT(namespace-envoy)
+
+extern "C" JNIEXPORT void JNICALL Java_io_envoyproxy_envoymobile_jni_JniHelperTest_getMethodId(
```

Comment:
Can we add a comment to this file which explains what it's doing? There are lot of thing defined in this file, but from the file name, I would have expected it to be a unit test.

Reply:
Done. Comment added.